contrib/mpi-proxy-split/lower-half/gethostbyname-static/Makefile

```diff
@@ -23,3 +23,8 @@ clean:
 dist: clean
 	dir=`basename $$PWD` && cd .. && tar zcvf ./$$dir.tar.gz ./$$dir
 	dir=`basename $$PWD` && ls -l ../$$dir.tar.gz
+
+install:
+	make gethostbyname_proxy
+	cp gethostbyname_proxy -f ../../../../bin
```

Comment:
I think this is not a general practice for `make install`
`make install` usually should copy those binaries to the system directory, by default is `/usr/local`, but could be configured by `--prefix` when `./configure`

It looks like other binaries for MANA, `lh_proxy` for example, also has the same problem, I think we can have a separate ticket to fix all of these.

And I would suggest don't use `make install` for development purpose, it looks like it will copy those `dmtcp` binaries to the system directory, which I don't think it's something you want to do in you development.

Reply:
> And I would suggest don't use make install for development purpose, it looks like it will copy those dmtcp binaries to the system directory, which I don't think it's something you want to do in you development.

I know what you mean by this, but it would seem like `make mana` calls a lot of `make install` instances in subdirectories that doesn't actually copy it to the system directory. This might be a misnomer - perhaps we could recategorize these instances of `install` as something else - maybe `mana`?

Reply:
Yes, I found `make mana` will `cd contrib/mpi-proxy-split && make install`, but that is not a right way to do. I think there is a common understanding of `make install` and we should follow the standard.

I'm fine with current version, as long as you know you only need to do `make mana`. And `make install` won't copy mana binaries to the right place, which I filed a ticket to track this issue.

If you check `dmtcp`, you will find that all dmptcp binaries will be generated in the `./bin` directory by just typing `make`, and it won't do `make install`, executing `make install` will copy them to the system directory(`/usr/local` by default, and configurable by `--prefix`) That's the way we should follow.

Reply:
okay. if there are no further issues, i'll merge this in.